.travis.yml

```diff
@@ -47,9 +51,18 @@ matrix:
   include:
     # Builds that are executed for every PR
     - os: osx # run base tests on both platforms
+      if: branch IN (auto, try)
     - os: linux
     - os: windows
       env: CARGO_INCREMENTAL=0 OS_WINDOWS=true
+    # i686 toolchain could run on x86_64 system.
+    - os: linux
+      env: HOST_TOOLCHAIN=i686-unknown-linux-gnu
+      addons:
+        apt:
+          packages:
+            - gcc-multilib
+      if: type = pull_request OR branch = master
```

Comment:
Why not also in `auto, try`?

Reply:
IMO clippy is portable enough. Only a few tests is needed on 32-bit pointer-wide systems.
So testing this on master or PR is enough.

Note that I change osx build job to only run on `auto` or `try` branch.
Because clippy is portable enough on macOS (I believe so).

Reply:
That's why, I would also only test this in `auto, try` and not in PRs or `master` (like osx). PR and master runs way more often, than auto or try

Reply:
But the opportunity of introduce breaking build on 32-bit systems is higher than on macOS.
So testing this early in PR helps wasting CI time.

Reply:
Yeah, but PRs which touch code/lints, that behave different for 32 and 64 bit systems are pretty rare. So this test should pass for most PRs anyway, so it would be cheaper to only run it on `auto, try`

Reply:
Fair enough. I will change it.

Reply:
Thanks!